CHANGELOG.txt

```diff
@@ -1,5 +1,20 @@
 == Changelog ==
 
+= 4.1.1 - 2020-05-19 =
+
+* Enhancement - Added notice about public uploads directory. #26207
+* Tweak - Disallow directory listing in woocommerce_uploads when "Redirect only" it's the selected download method. #26399
+* Fix - Added correct handling of nonces to database update notice dismissal. #26500
+* Dev - Updated WooCommerce admin version to 1.1.3 and Action Scheduler to 3.1.6.
+* Dev - Add prop `isEnabled` and a function to dynamically enable tracks. #26493
+
+**WooCommerce Admin**
+* Tweak - Onboarding: Add Jetpack flow back to onboarding profiler. #4382
+* Fix - Respect tracking opt-in before new page load. #4368
+
+**ActionScheduler**
+* Fix - Shutdown deprecated notice changed to a warning when as_* functions called without data store initialization. #546
+
```

Comment:
Changelog for 4.1.0 is missing, can you please add it as well?

Reply:
Yes, I'm not sure how I missed it.